cub/docs/gen_docs.bash

```diff
@@ -18,7 +18,7 @@ if [ ! -d cubimg ]; then
 fi
 
 if [ ! -n "$(find img -name 'example_range.png')" ]; then
```

Comment:
suggestion: you are searching in img instead of cubing, so the wget always happens
```suggestion
if [ ! -n "$(find cubimg -name 'example_range.png')" ]; then
```